### PR TITLE
[REFACTOR] Custom Fonts improvements

### PR DESCRIPTION
**Pull request by sfernandez11**
## Summary
* Added Montserrat font as default.
* Link assets after pod install.
* Refactor on CustomText and default styles.
* Added `baseFont` as navigation header font too.
* Added base fonts for italic, semiBold and bold CustomText variants

### Example code
```
<CustomText>Montserrat Regular</CustomText>
<CustomText italic>Montserrat Regular Italic</CustomText>
<CustomText semiBold>Montserrat SemiBold</CustomText>
<CustomText bold>Montserrat Bold</CustomText>
```
### Android
Captura de Pantalla 2019-10-02 a la(s) 17 23 48

### Ios
Captura de Pantalla 2019-10-02 a la(s) 17 22 41

## Trello Card
https://trello.com/c/0qKFzygU/135-actualizar-implementacion-de-custom-fonts-a-rn-060

### CODE REVIEW

File: generators/app/files.js [hunk 1]
```diff
@@ -21,9 +25,13 @@ module.exports.INDEX = 'index.js';
...
-// TESTING FILES
-module.exports.TESTS = '__tests__';
-module.exports.MOCKS = '__mocks__';
+// FONTS
+module.exports.MONTSERRAT_BOLD = `${module.exports.FONTS_PATH}/Montserrat-Bold.ttf`;
+module.exports.MONTSERRAT_BOLD_ITALIC = `${module.exports.FONTS_PATH}/Montserrat-BoldItalic.ttf`;
+module.exports.MONTSERRAT_ITALIC = `${module.exports.FONTS_PATH}/Montserrat-Italic.ttf`;
+module.exports.MONTSERRAT_SEMIBOLD = `${module.exports.FONTS_PATH}/Montserrat-SemiBold.ttf`;
+module.exports.MONTSERRAT_SEMIBOLD_ITALIC = `${module.exports.FONTS_PATH}/Montserrat-SemiBoldItalic.ttf`;
+module.exports.MONTSERRAT = `${module.exports.FONTS_PATH}/Montserrat.ttf`;
```
Comment (rodrigoWolox) [inline generators/app/files.js]: Can we declare a const for the string `${module.exports.FONTS_PATH}`?

File: generators/app/templates/src/app/components/CustomText/styles.js [hunk 2]
```diff
@@ -1,46 +1,46 @@
...
 import { blue, white, green, gray, transparent } from '@constants/colors';
-import { IS_SMALL_DEVICE } from '@constants/platform';
+import { SIZES } from '@constants/fonts';
+import { moderateScale } from '@utils/scalingUtils';
 
-const amountToResize = 2 * IS_SMALL_DEVICE;
+const getColors = colorsObj =>
+  Object.keys(colorsObj)
+    .map(colorName => ({ [colorName]: { color: colorsObj[colorName] } }))
+    .reduce((colors, color) => ({ ...colors, ...color }), {});
```
Comment (rodrigoWolox) [inline generators/app/templates/src/app/components/CustomText/styles.js]: We can use just reduce here i think.
```
Object.keys(colorsObj)
    .reduce((colors, color) => ({ ...colors, ...{ [color]: { color: colorsObj[color] }  }), {});
```

File: generators/app/templates/src/app/components/CustomText/styles.js [hunk 3]
```diff
@@ -1,46 +1,46 @@
...
+
+const getSizes = sizesObj =>
+  Object.keys(sizesObj)
+    .map(sizeName => ({ [sizeName]: { fontSize: moderateScale(sizesObj[sizeName]) } }))
+    .reduce((sizes, size) => ({ ...sizes, ...size }), {});
```
Comment (rodrigoWolox) [inline generators/app/templates/src/app/components/CustomText/styles.js]: Same as before

File: generators/app/files.js (hunk 1 above)
Comment (WFolini) [inline generators/app/files.js]: I think that it isn't necessary to add all font files. It is just fine if you declare the folder path. All fonts will be imported as well

File: generators/app/tasks/installPods.js [hunk 4]
```diff
@@ -4,8 +4,8 @@ module.exports = function configurePods() {
   return runCommand({
     command: ['pod', ['install'], { cwd: `${process.cwd()}/${this.projectName}/ios` }],
     loadingMessage: 'Installing Pods...',
+    successMessage: 'Pods ready!',
+    failureMessage: 'Pod install failed. Turn verbose mode on for detailed logging',
     context: this.options
-  }).then(({ spinner }) => {
```
Comment (WFolini) [inline generators/app/tasks/installPods.js]: 👌

**Pull request merged**